packages/@aws-cdk/aws-stepfunctions-tasks/lib/emrcontainers/utils/role-policy/index.py

```diff
@@ -0,0 +1,15 @@
+import subprocess as sp
+import os
+import logging
+
+#https://github.com/aws/aws-cdk/tree/master/packages/%40aws-cdk/aws-stepfunctions#custom-state
+def handler(event, context):
```

Comment:
This lambda function is not aware of cloudformation lifecycle events; it will perform the same action on `create`, `update`, and `delete`. In an ideal world, we would want to be able to delete the role trust policy on `delete`, but there is no cli api to call for that, so I am happy if we just do nothing on `delete`.

What that means is that we should be only calling the command for `event['RequestType'] != 'Delete'`.